docs-site/docs/configuration/authentication.md

```diff
@@ -0,0 +1,109 @@
+---
+sidebar_position: 1
+id: authentication
+title: Connecting to an Authentication Server
+---
+
+## Setup
+
+### Enable remote authentication
+
+By default, remote authentication is disabled. To enable it you will need to edit the config file `config/auth_config.yml`
+
+It looks like this.
+
```

Comment:
**question:** You committed the `config/auth_config.yml` file to look like this already.  Did you mean to ?

Reply:
Yes, though it might make more sense to have the lines commented out to start.

Reply:
I commented out the inactive lines and updated the docs to reflect un-commenting and editing the config file.